Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java

```diff
@@ -39,12 +40,16 @@
 import org.kitodo.data.database.beans.User;
 import org.kitodo.production.services.ServiceManager;
 import org.kitodo.production.services.data.ProcessService;
+import org.kitodo.utils.ProcessTestUtils;
```

Comment:
Perhaps create a dedicated package for test-specific components?

Reply:
Hm, this is already the only class in its package. Or did I miss your point?

Reply:
My thought was whether we could create a subpackage named 'test' for the test framework.